WIP 3.0 Classes/Mage - Reworked.md

```diff
@@ -0,0 +1,1152 @@
+- **Components:** V, S
+- **Duration:** Instantaneous
+___
+You form a large spike of ice above your head, and launch it at a creature that you can see within range, causing it to shatter across their body. The target must make a Dexterity saving throw. It takes 8d8 + 40 cold damage on a failed save, or half as much damage on a successful one. 
+
+If this damage reduces a creature to 0 hit points, it is reduced to shards of ice, leaving behind everything it was wearing and carrying. The creature can be restored to life only by means of a true resurrection or a wish spell.
+
+#### Ice Block
+*4th-level abjuration*
+___
+- **Casting Time:** 1 reaction, when you take damage
+- **Range:** Self
+- **Components:** V, S
+- **Duration:**  Concentration, up to 1 minute
+___
+Ice rapidly materializes around you, protecting you from harm. You gain 40 temporary hit points and immunity to cold damage for the spells duration. While the spell is active you can't move and take any actions or reactions. The spell remains until the temporary hit points have been expend or until your concentration breaks.
+
+#### Ice Nova
+*8th-level evocation*
+___
+- **Casting Time:** 1 action
+- **Range:** 120 feet
+- **Components:** V, S, M (glass dust and water)
+- **Duration:** Instantaneous
+___
+Frigid cold rapidly expands in a 15-foot-radius centered on a point you you can see within range. Each creature in the cold must succeed on a Dexterity saving throw. On a failed save, a creature takes 10d6 cold damage and is restrained for 1 minute as ice encases the lower part of their legs. On a successful save, it takes half as much damage and isn't restrained by this spell.
+
+A creature restrained by the ice can use its action to make a Strength check against your spell save DC. On a success, it frees itself. Alternatively can the ice be broken by dealing a total of 40 hit points worth of damage to it, its armor class is 10. 
+
+#### Jaina's Flying Ship
+*9th-level transmutation*
+___
+- **Casting Time:** 1 minute
+- **Range:** Touch
+- **Components:** V, S
+- **Duration:** 24 hours
+___
+You channel your powers into a nonmagical ship within range, the ship can be no larger than 100 feet long, and 30 feet wide. For the spells duration, the ship gains a flying speed of 60 feet, the ship changes direction and altitude at your command, continuing its course until a new command is given or the spells duration ends.
+
+The flying ship has an Armor Class of 15, and 300 hit points; immunity to poison and psychic damage, and immunity to all conditions.
+
+<div class='footnote'>PART 2 | SPELLS </div>
+
+\pagebreakNum
+
+It has a maximum carrying capacity of 50 tons (limited by the size of the ship itself). It is not slowed by any weight until its maximum capacity is reached, at which point the ship loses its flying property and falls at a rate of 60 feet per round until it hits the ground. Taking no falling damage from the fall, and leaving its passengers unharmed.
```

Comment:
The last part should probably read, "It takes no falling damage from the fall, and its passengers are unharmed."